help/CreateImageAndAzureResources.md

```diff
@@ -1,11 +1,122 @@
+```
+
+Import [GenerateResourcesAndImage](helpers/GenerateResourcesAndImage.ps1) script from `/helpers` folder, and run `GenerateResourcesAndImage` function via Powershell.
+
+```
+cd C:\virtual-environments
+
+Import-Module "helpers\GenerateResourcesAndImage.ps1"
+
+GenerateResourcesAndImage -SubscriptionId {YourSubscriptionId} -ResourceGroupName "myTestResourceGroup" -ImageGenerationRepositoryRoot "$pwd" -ImageType Ubuntu1604 -AzureLocation "East US"
+```
+Where:
+- `SubscriptionId` - The Azure subscription Id where resources will be created.
+- `ResourceGroupName` - The Azure resource group name where the Azure resources will be created.
+- `ImageGenerationRepositoryRoot` - The root path of the image generation repository source.
+- `ImageType` - The type of the image being generated. Valid options are: "Windows2016", "Windows2019", "Ubuntu1604", "Ubuntu1804", "Ubuntu2004".
+- `AzureLocation` - The location of the resources being created in Azure. For example "East US".
+
+The function automatically creates all required Azure resources and kicks off packer image generation for the selected image type.
+
+*Please, check synopsis of `GenerateResourcesAndImage` for details about non-mandatory parameters.*
+
+#### Generated VM Deployment
+After the successful image generation, Virtual Machine can be created from the generated VHD using [CreateAzureVMFromPackerTemplate](helpers/CreateAzureVMFromPackerTemplate.ps1) script.
+
+```
+cd C:\virtual-environments
+
+Import-Module helpers\CreateAzureVMFromPackerTemplate.ps1
+
+CreateAzureVMFromPackerTemplate -SubscriptionId {YourSubscriptionId}  -ResourceGroupName {ResourceGroupName} -TemplateFile "C:\BuildVmImages\temporaryTemplate.json" -VirtualMachineName "testvm1" -AdminUsername "shady1" -AdminPassword "SomeSecurePassword1" -AzureLocation "eastus"
+```
+Where:
+- `SubscriptionId` - The Azure subscription Id where resources will be created.
+- `ResourceGroupName` - The Azure resource group name where the Azure virtual machine will be created.
+- `TemplateFilePath` - The path to the json ARM-template generated by packer during image generation locally.* 
+- `VirtualMachineName` - The name of the virtual machine to be generated.
+- `AdminUserName` - The administrator username for the virtual machine to be created.
+- `AdminPassword` - The administrator password for the virtual machine to be created.
+- `AzureLocation` - The location where the Azure virtual machine will be provisioned. Example: "eastus"
+
+\* *ARM-template can be obtained from the Packer output. For now, it seems like there is an [Az CLI bug](https://github.com/Azure/azure-cli/issues/5899) with specifying the template through a URI, so download the template from URI, that will be provided at the bottom of image-generation log, and use the local path of the template file.*
+
+The function creates an Azure VM from a template and also generates network resources in Azure to make the VM accessible.
```

Comment:
Usually, `also` is not used at the middle of sentense

Reply:
Thanks! Fixed.